src/StepInput/StepInput.js

```diff
@@ -0,0 +1,132 @@
+import Button from '../Button/Button';
+import classnames from 'classnames';
+import { FORM_MESSAGE_TYPES } from '../utils/constants';
+import FormInput from '../Forms/FormInput';
+import FormMessage from '../Forms/_FormMessage';
+import InputGroup from '../InputGroup/InputGroup';
+import keycode from 'keycode';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
+
+const StepInput = React.forwardRef(({
+    children,
+    className,
+    disabled,
+    disableStyles,
+    placeholder,
+    readOnly,
+    validationState,
+    value,
+    ...rest
+}, ref) => {
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/input.css');
```

Comment:
We can remove this because `FormInput` will bring input.css with it

Reply:
Done.Please review.